### PR TITLE
Attribute inv() added in matexpr.py

### LINKED ISSUE #13541: Bugs inverting a matrix of matrix elements
```pytb
>>> Matrix(MatrixSymbol('b', 3, 3)).inv()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "./sympy/matrices/matrices.py", line 2797, in inv
    return self._eval_inverse(**kwargs)
  File "./sympy/matrices/dense.py", line 260, in _eval_inverse
    rv = M.inverse_GE(iszerofunc=iszerofunc)
  File "./sympy/matrices/matrices.py", line 2710, in inverse_GE
    red = big.rref(iszerofunc=iszerofunc, simplify=True)[0]
  File "./sympy/matrices/matrices.py", line 889, in rref
    normalize_last=normalize_last)
  File "./sympy/matrices/matrices.py", line 544, in _eval_rref
    zero_above=True)
  File "./sympy/matrices/matrices.py", line 660, in _row_reduce
    get_col(piv_col)[piv_row:], iszerofunc, simpfunc)
  File "./sympy/matrices/matrices.py", line 4130, in _find_reasonable_pivot
    if x.equals(S.Zero):
  File "./sympy/core/expr.py", line 657, in equals
    constant = diff.is_constant(simplify=False, failing_number=True)
  File "./sympy/core/expr.py", line 606, in is_constant
    deriv = expr.diff(w)
  File "./sympy/core/expr.py", line 2967, in diff
    return Derivative(self, *new_symbols, **assumptions)
  File "./sympy/core/function.py", line 1137, in __new__
    if v._diff_wrt:
AttributeError: 'MatrixSymbol' object has no attribute '_diff_wrt'
```

Also, I would expect both of these to work

```pytb
>>> MatrixSymbol('b', 3, 3).inv()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'MatrixSymbol' object has no attribute 'inv'
>>> (MatrixSymbol('b', 3, 3)**-1).as_explicit()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "./sympy/matrices/expressions/matexpr.py", line 299, in as_explicit
    for i in range(self.rows)])
  File "./sympy/matrices/expressions/matexpr.py", line 299, in <listcomp>
    for i in range(self.rows)])
  File "./sympy/matrices/expressions/matexpr.py", line 298, in <listcomp>
    for j in range(self.cols)]
  File "./sympy/matrices/expressions/matexpr.py", line 248, in __getitem__
    return self._entry(i, j)
  File "./sympy/matrices/expressions/matpow.py", line 47, in _entry
    + " of matrix power either not defined or not implemented")
NotImplementedError: (0, 0) entry of matrix power either not defined or not implemented
```

### PR DESCRIPTION
**Pull request by vishalg2235**
Fixes #13541  
A new attribute function `inv` is added in matexpr.py which return `self**-1`. And, test case added.
Ping @gxyd @asmeurer

### CODE REVIEW

File: sympy/matrices/expressions/matexpr.py [hunk 1]
```diff
@@ -261,6 +261,9 @@ def transpose(self):
     def inverse(self):
         return self._eval_inverse()
 
+    def inv(self):
```
Comment (gxyd) [inline sympy/matrices/expressions/matexpr.py]: How about setting `inv = inverse`?

**Comment by ylemkimon** [comment]
`inverse` calls `_eval_inverse` which calls `Inverse` and `**-1` calls `__pow__` which calls `Inverse`. So the result is same.

**Comment by vishalg2235** [comment]
@ylemkimon can you brief me what was desired thing/functionality ? I thought I have to make API of Matrix and MatrixExpr consistent.

**Comment by ylemkimon** [comment]
You can just do `inv = inverse`.

**Review by ylemkimon — APPROVED** [review]

**Pull request merged**